docs/content/v2/en/options.md

```diff
@@ -7,12 +7,16 @@ category: Guide
 ---
 
 ## `crossorigin`
-- Type: `String`
+- Type: `String`, `Boolean`
   - Default: `'anonymous'`
-  - String values: `'anonymous'`, `'use-credentials'` or `undefined` [learn more](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)
+  - Valid values: `anonymous`, `use-credentials`, `''`, `true`, `false`
```

Comment:
what will happen, when I assign true to crossorigin?
valid values are `anonymous`, `use-credentials`, `''`, `false`

Reply:
`true` is in this analog to `''` default value `anonymous`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin

This allows activation and deactivation via `Boolean`.